src/hoplon/firebase/auth.cljs.hl

```diff
@@ -51,6 +51,7 @@
 (defn- handle-login
   [auth & [ref default]]
   (let [ref (when ref (fbdb/get-in ref (fbuser/uid auth)))]
+    (when (nil? @*user-auth*) (reset! *user-auth* auth))
```

Comment:
Do we need to check if `*user-auth*` is nil or should we just set it again anyway? Also on page reload does this mean `*user-cred*` is nil at this point too?

Also should this whole `let` body be wrapped in a `dosync`?